src/sh/python-lint/format.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# cspell:words toplevel
+ROOT_DIR=$(git rev-parse --show-toplevel)
+$ROOT_DIR/src/sh/python-lint/./run-in-poetry-subdir.sh "poetry run python -m format"
```

Comment:
- [ ] Note this should run poetry check first

Reply:
Done!